torch/csrc/jit/pass_manager.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <torch/csrc/jit/ir.h>
+
+/* `getPasses()` returns a vector of passes that will be executed after
+ * differentiation but before any fusion.  This is the de-facto location
```

Comment:
I wonder if it would make sense to generalize it a little and introduce 1) target-independent and 2) target-specific passes. The fusion would be in (2), but I expect more stuff than just fusion to happen there down the road.

Reply:
Agreed, but I think we should push it to a later PR.  I believe there will be some other considerations when it comes to the hierarchy/order of pass invocations.